msfdb

```diff
@@ -204,28 +187,30 @@ end
 
 def stop_db
   update_db_port
-
-  if run_cmd("pg_ctl -o \"-p #{@options[:db_port]}\" -D #{@db} status") == 0
-    puts "Stopping database at #{@db}"
-    run_cmd("pg_ctl -o \"-p #{@options[:db_port]}\" -D #{@db} stop")
-  else
-    puts "Database is no longer running at #{@db}"
-  end
+  @db_driver.stop_db
 end
 
 def restart_db
-  stop_db
-  start_db
+  @db_driver.restart_db
 end
 
 def create_db
-  puts "Creating database at #{@db}"
-  Dir.mkdir(@db)
-  run_cmd("initdb --auth-host=trust --auth-local=trust -E UTF8 #{@db}")
+  @db_driver.init_db
+end
 
-  File.open("#{@db}/postgresql.conf", 'a') do |f|
-    f.puts "port = #{@options[:db_port]}"
-  end
+def create_db_users
+  puts 'Creating database users'
+  run_psql("create user #{@options[:msf_db_user]} with password '#{@msf_pass}'")
+  run_psql("create user #{@options[:msftest_db_user]} with password '#{@msftest_pass}'")
+  run_psql("alter role #{@options[:msf_db_user]} createdb")
+  run_psql("alter role #{@options[:msftest_db_user]} createdb")
+  run_psql("alter role #{@options[:msf_db_user]} with password '#{@msf_pass}'")
+  run_psql("alter role #{@options[:msftest_db_user]} with password '#{@msftest_pass}'")
```

Comment:
Just confirming, should this interact with the driver object? 👀

Reply:
Nope! anything `psql` should be consistent across all types of database setups